### ISSUE TITLE
/v2/recipes: remove output_item_id when output_upgrade_id is present.

### ISSUE THREAD

**Issue by lye** [arenanet/api-cdi#209]
Scribe recipes that create guild upgrades don't actually produce output items (the value returned in `output_item_id` is a mostly bogus value).

**Comment by queicherius** [comment]
I would like to chime in against this and propose to instead include the item id in the `/v2/guild/upgrades` endpoint. The reason for it is that it allows for easier recipe generation. An example:

1. I can look at the recipe for guild hero banners: https://api.guildwars2.com/v2/recipes/11563
2. I see that the upgrade id `93` is needed
3. I can call https://api.guildwars2.com/v2/guild/upgrades/93 to get the item id (**currently not possible, item id missing**)
4. I can search for the recipe for the upgrade via https://api.guildwars2.com/v2/recipes/search?output=75869 (already possible, here item id of the guild hero banner)
5. I call the new recipe endpoint, etc, etc. until I have a recursive tree

Of course you can go around this by just crawling all recipes first and then filtering them by `output_upgrade_id`, but I just wanted to give another perspective.

**Comment by lye** [comment]
So, my understanding of the guild upgrade system is that *originally* the upgrades weren't deposited in the guild inventory -- instead it leveraged the existing crafting code and put an "Immediate" item in your inventory which was immediately consumed and added to your guild inventory. At some point that system was gutted (but some dregs remain, such as an `output_item_id` for scribe recipes). AFAIK with the released system, that field is totally unused and only exists because the data was designed with old constraints.

My guess is that future guild upgrade recipes will not necessarily have an `output_item_id` (or will have a bogus one). I can add it in, but can't guarantee that the data won't be totally meaningless (e.g., all future `output_item_ids` for scribe recipes that deposit to the guild inventory may be `Stick of Butter` -- the game code will totally ignore it).

**Comment by queicherius** [comment]
Oooh if that's the case then it would make sense to remove it since it would break on newer one's anyway. I thought the immediate item would still exist.

**Comment by lye** [comment]
Yeah. The immediate/consumable items *kind of* still exist for some existing recipes, but that's an implementation artifact. The code that handles the scribe crafting ignores them entirely.

The guild upgrades which can be "consumed" to give you an item (e.g., wvw siege blueprints, maybe the banners (might be misreading the banners)) can definitely be fixed to emit the `item_id` in `/v2/guild/upgrades` though.
